packages/py-moose-lib/moose_lib/streaming/streaming_function_runner.py

```
@@ -673,6 +695,11 @@ def process_messages():
     def shutdown(signum, frame):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `shutdown`

Add return type annotation: `None`

(ANN202)